### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by domhnallmorr** [domhnallmorr/Pitwall_Flet#4] [Announcements]
General discussion for anyone who wants to leave a comment.

**Comment by felipedemelobarbosa** [comment]
It would be good to be able to check drivers ratings while in contract negociation. As of now, it is only possible to see age and desired contract length. 


![image](https://github.com/user-attachments/assets/9d72f8f4-a617-4a5f-9e99-b56b3a29c233)

**Comment by felipedemelobarbosa** [comment]
Also, it seems that, after a driver refuses a contract offer, it is not possible to make a new offer.

**Comment by domhnallmorr** [comment]
Thanks for the comments. Agreed on both. For the 2nd point, not being able to make a 2nd offer, that is deliberate for the moment. I wanted to setup a system to track which drivers were approached, to avoid the player spamming offers at 1 particular driver. 

preventing the player from approaching the driver again was a simple way to check this in the UI. At some point I will add the ability to make a counter offer (1 week later).

Thanks again.
